content/hi/docs/setup/production-environment/container-runtimes.md

```diff
@@ -0,0 +1,407 @@
+```
+
+कॉन्फ़िगरेशन को लागू करने के लिए, नोड को रीबूट करना आवश्यक है।
+
+cgroup v2 में स्विच करते समय उपयोगकर्ता अनुभव में कोई उल्लेखनीय अंतर नहीं होना चाहिए, जब तक कि उपयोगकर्ता सीग्रुप फाइल सिस्टम को सीधे नोड पर या कंटेनरों के भीतर से एक्सेस कर रहे हैं। 
+
+इसका उपयोग करने के लिए, cgroup v2 को CRI रनटाइम द्वारा भी समर्थित होना चाहिए। 
+
+### Kubeadm प्रबंधित क्लस्टर में `systemd` ड्राइवर में माइग्रेट करना
+
+यदि आप मौजूदा kubeadm प्रबंधित क्लस्टर में `systemd` cgroup ड्राइवर में माइग्रेट करना चाहते हैं, तो [माइग्रेशन गाइड](/docs/tasks/administer-cluster/kubeadm/configure-cgroup-driver/) का पालन करें।
+
+## Container runtimes
+
+{{% thirdparty-content %}}
+
+### containerd
+
+यह खंड कंटेनरड को सीआरआई रनटाइम के रूप में उपयोग करने के लिए आवश्यक कदम बताता है।
+अपने सिस्टम पर containerd स्थापित करने के लिए निम्नलिखित कमांड का उपयोग करें:
+पूर्वापेक्षाएँ स्थापित और कॉन्फ़िगर करें:
+
+```shell
+cat <<EOF | sudo tee /etc/modules-load.d/containerd.conf
+overlay
+br_netfilter
+EOF
+
+sudo modprobe overlay
+sudo modprobe br_netfilter
+
+# Setup required sysctl params, these persist across reboots.
+cat <<EOF | sudo tee /etc/sysctl.d/99-kubernetes-cri.conf
+net.bridge.bridge-nf-call-iptables  = 1
+net.ipv4.ip_forward                 = 1
+net.bridge.bridge-nf-call-ip6tables = 1
+EOF
+
+# Apply sysctl params without reboot
+sudo sysctl --system
+```
+
+कंटेनर स्थापित करें:
+
+{{< tabs name="tab-cri-containerd-installation" >}}
+{{% tab name="Linux" %}}
+
+1. आधिकारिक डॉकर रिपॉजिटरी से `containerd.io` पैकेज स्थापित करें।  
+अपने संबंधित लिनक्स वितरण के लिए डॉकर रिपॉजिटरी स्थापित करने और `containerd.io` पैकेज स्थापित करने के निर्देश यहां देखे जा सकते हैं
+[Install Docker Engine](https://docs.docker.com/engine/install/#server).
+
+2. कंटेनर को कॉन्फ़िगर करें:
+
+   ```shell
+   sudo mkdir -p /etc/containerd
+   containerd config default | sudo tee /etc/containerd/config.toml
+   ```
+
+3. कंटेनर को पुनरारंभ करें:
+
+   ```shell
+   sudo systemctl restart containerd
+   ```
+
+{{% /tab %}}
+{{% tab name="Windows (PowerShell)" %}}
+
+Powershell सत्र प्रारंभ करें, `$Version` को वांछित संस्करण पर सेट करें (उदाहरण: `$Version=1.4.3`), और फिर निम्न आदेश चलाएँ: 
+
+1. कंटेनरड डाउनलोड करें: 
+
+   ```powershell
+   curl.exe -L https://github.com/containerd/containerd/releases/download/v$Version/containerd-$Version-windows-amd64.tar.gz -o containerd-windows-amd64.tar.gz
+   tar.exe xvf .\containerd-windows-amd64.tar.gz
+   ```
+
+2. एक्सट्रेक्ट एंड कॉन्फ़िगर:
+
+   ```powershell
+   Copy-Item -Path ".\bin\" -Destination "$Env:ProgramFiles\containerd" -Recurse -Force
+   cd $Env:ProgramFiles\containerd\
+   .\containerd.exe config default | Out-File config.toml -Encoding ascii
+
+   # Review the configuration. Depending on setup you may want to adjust:
+   # - the sandbox_image (Kubernetes pause image)
+   # - cni bin_dir and conf_dir locations
+   Get-Content config.toml
+
+   # (Optional - but highly recommended) Exclude containerd from Windows Defender Scans
```

Comment:
nit: localize this comment

Reply:
How can I localize the comments? these are the commands, they needs to be typed exactly right?

Reply:
okay @sftim